src/models/TranslationValue/addTranslationValueToTranslationKey/index.js

```diff
@@ -75,10 +81,11 @@ const addTranslationValueToTranslationKey = async ({
   // Saving key
   try {
     const translationValue = new TranslationValue(value, locale)
+    translationKey.translationValues = []
```

Comment:
isn’t it already an empty array if you don’t fetch them?

Reply:
You could also have: `translationKey.translationvalues = [translationValue]` instead of pushing it later.

Reply:
It's actually undefined if we don't fetch them (empty array if we fetch them but there are none)